src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,12 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparingInt(String::length).reversed())
+                .distinct()
+                .map(String::toLowerCase)
+                .forEach(System.out::println);
```

Comment:
100개의 단어를 추출해야한다는 요구사항도 반영해주시면 좋을 것 같아요 :)
```suggestion
        words.stream()
                .filter(word -> word.length() > 12)
                .sorted(Comparator.comparingInt(String::length).reversed())
                .distinct()
                .limit(100)
                .map(String::toLowerCase)
                .forEach(System.out::println);
```